source/site-administrators/studio/rte2-setup.rst

```diff
@@ -148,6 +163,112 @@ You can also add custom elements to the rule set and can be done by simply addin
       :width: 85%
       :align: center
 
+   |
+
+.. _adding-external-plugins:
+
+^^^^^^^^^^^^^^^^^^^^^^^
+Adding External Plugins
+^^^^^^^^^^^^^^^^^^^^^^^
+
+TinyMCE provides an option to specify URLS to plugins outside the tinymce plugins directory.  These external plugins allow the user to extend TinyMCE.  For example, you can create custom dialogs, buttons, menu items, etc.
+
+For more information on the Tiny MCE external_plugins option, see https://www.tiny.cloud/docs/configure/integration-and-setup/#external_plugins
+
+The Crafter Studio user does not have full control of the tinymce initialization.  To add a custom button to the toolbar in Crafter Studio, it would be done using the external plugin route since, what TinyMCE docs advise is not viable in Studio without doing a whole form control plugin where they'd have full control of tinymce initialization.
```

Comment:
```suggestion
The Crafter Studio developer does not have full control of the tinymce initialization.  To add a custom button to the toolbar in Crafter Studio, it would be done using the external plugin route since, what TinyMCE docs advise – i.e. using the `setup` function to add the button – is not viable in Studio without creating an entire form control plugin where they'd have full control of tinymce initialization.
```

Please link "form control plugin" to the docs for creating form control plugins